builds/e2e/isa-95-smoke-test.yaml

```diff
@@ -0,0 +1,245 @@
+trigger: none
+pr: none
+
+variables:
+  itProxy: http://10.16.8.4:3128
+  otProxy: http://10.16.5.4:3128 
+  ressourceGroup: nested-edge-isa95 
+
+resources:
+  pipelines:
+  - pipeline: images
+    source: 'Azure-IoT-Edge-Core Build Images'
+    branch: master
+  - pipeline: packages
+    source: 'Azure-IoT-Edge-Core Edgelet Packages'
+    branch: master
+
+stages:
+- stage: SetupVM
+  jobs:
+  - job: SetupVM_level5
+    displayName: SettingUp level 5
+    timeoutInMinutes: 180
+    variables:
+      artifactName: iotedged-ubuntu18.04-amd64
+      identityServiceArtifactName: packages_ubuntu-18.04_amd64
+      identityServicePackageFilter: aziot-identity-service_*_amd64.deb
+      vsts.project: $(System.TeamProjectId)
+      level: '5'
+      deploymentFile: 'nestededge_topLayerBaseDeployment.json'
```

Comment:
>'nestededge_topLayerBaseDeployment.json' [](start = 22, length = 40)

shouldn't this be _amqp?  I thought golden path was suppose to be without broker.

Reply:
Fixed